webdriver-spec.html

```diff
@@ -459,7 +460,15 @@
   <ul>
    <li><dfn data-lt="supported property index"><a href=http://heycam.github.io/webidl/#dfn-supported-property-indices>Supported property indices</a></dfn>
    <li><dfn><a href="https://heycam.github.io/webidl/#syntaxerror"><code>SyntaxError</code></a></dfn></li>
+   <!-- DOMException --><li><a href="https://heycam.github.io/webidl/#dfn-DOMException"><dfn><code>DOMException</code></dfn></a>
   </ul>
+
+  <dt>XPATH
+  <dd><p>The following terms are defined in the Document Object Model XPath standard [[!XPATH]]
```

Comment:
These are not in the DOM spec, the DOM Spec doesnt deal with XPath as far as I can see.